### ISSUE TITLE
[TW-926] Support journalling of entries

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#961]
_Paul Beckingham on 2009-05-25T21:30:24Z says:_ 

 Or at least support the generation of a journal from the undo stack.  Similar to a shadow file.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2009-05-25T21:30:24Z
Modified: 2014-02-09T02:11:52Z
```

**Comment by taskwarrior** [comment]
_David Patrick on 2009-12-02T12:16:51Z says:_ 

 this function should be renamed, or hidden, to avoid confusion with "task journal"

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2011-03-27T10:29:16Z says:_ 

 Implemented in the 1.9.4 'info' command.
